### PR TITLE
Chi is no longer with us :(

### PR DESCRIPTION
**Pull request by kvmware**

### CODE REVIEW

File: working-groups/WORKING-GROUPS.md [hunk 1]
```diff
@@ -390,3 +389,9 @@ Except as otherwise noted, the content of this page is licensed under the
 [Creative Commons Attribution 4.0 License](https://creativecommons.org/licenses/by/4.0/),
 and code samples are licensed under the
 [Apache 2.0 License](https://www.apache.org/licenses/LICENSE-2.0).
+
+## Productivity
+| &nbsp;                                                    | Emeritus Leads       | Profile                                   | Duration  |
+| --------------------------------------------------------- | -------------------- | ----------------------------------------- | --------- |
+| <img width="30px" src="https://github.com/chizhg.png">    | Chi Zhang            | [chizhg](https://github.com/chizhg)       | ?-2022    |
```
Comment (dprotaso) [inline working-groups/WORKING-GROUPS.md:396]: line 320 has a list of existing emeritus productivity leads - we can add chi to that

Comment (kvmware) [reply working-groups/WORKING-GROUPS.md]: my bad i missed that. Fixed

File: working-groups/WORKING-GROUPS.md [hunk 2]
```diff
@@ -313,12 +313,12 @@ performance/scale/load testing infrastructure
 
 | &nbsp;                                                   | Leads            | Company   | Profile                                 |
 | -------------------------------------------------------- | ---------------- | --------- | --------------------------------------- |
-| <img width="30px" src="https://github.com/chizhg.png">   | Chi Zhang        | Google    | [chizhg](https://github.com/chizhg)     |
 | <img width="30px" src="https://github.com/kvmware.png">  | Krsna Mahapatra  | VMware    | [kvmware](https://github.com/kvmware)   |
 | <img width="30px" src="https://github.com/upodroid.png">  | Mahamed Ali     | Rackspace Technology | [upodroid](https://github.com/upodroid) |
 
 | &nbsp;                                                    | Emeritus Leads | Profile                                   | Duration  |
 | --------------------------------------------------------- | -------------- | ----------------------------------------- | --------- |
+| <img width="30px" src="https://github.com/chizhg.png">    | Chi Zhang      | [chizhg](https://github.com/chizhg)       | ?-2022    |
```
Comment (dprotaso) [inline working-groups/WORKING-GROUPS.md:321]: ref: https://github.com/knative/community/pull/121

```suggestion
| <img width="30px" src="https://github.com/chizhg.png">    | Chi Zhang      | [chizhg](https://github.com/chizhg)       | 2020-2022    |
```

**Pull request merged**